src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.ts

```diff
@@ -65,11 +82,14 @@ export class NotificationPopupComponent implements OnInit {
     private addNotification(notification: Notification): void {
         // Only add a notification if it does not already exist.
         if (notification && !this.notifications.some(({ id }) => id === notification.id)) {
-            // For now only notifications about a started quiz should be displayed.
             if (notification.title === 'Quiz started') {
                 this.addQuizNotification(notification);
                 this.setRemovalTimeout(notification);
             }
+            if (notification.title === LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE) {
```

Comment:
Why two equal signs in some places and three here?

Reply:
I fixed it.